src/status_im/chat/screen.cljs

```diff
@@ -92,7 +92,7 @@
         [touchable-highlight
          {:on-press #(dispatch [:set-chat-ui-props {:show-actions? false}])}
          [view st/action
-          [icon :up st/up-icon]]]
+          [vi/icon :icons/dropdown_up]]]
```

Comment:
If we follow the style guidelines it should be vector-icon instead of vi. I wouldn't be totally opposed to visit if we have a consensus on when to use shorthand but I need to know because in my refactoring I started to rename rn into react-native for instance.

Reply:
@yenda Please don't rename `rn` to `react-native`, it's pretty established now

Reply:
@yenda Rule probably is: if you change something that is already used we should discuss it. aka there is no rule yet.

Reply:
after all icons will be replaced on svg we will move icon into react ns, so maybe we can leave vi for now because it will be renamed later